docs/resources/obs_bucket_acl.md

```diff
@@ -0,0 +1,119 @@
+---
+subcategory: "Object Storage Service (OBS)"
+---
+
+# huaweicloud_obs_bucket_acl
+
+Manages an OBS bucket acl resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "bucket" {}
+variable "owner_permission" {
+  type = object({
+    access_to_bucket = list(string)
+    access_to_acl    = list(string)
```

Comment:
it's hard to understanding, please use a specific value instead of define a variable.